### ISSUE TITLE
Clicking the overview tab leads to error reading from database

### ISSUE THREAD

**Issue by mrkskwsnck** [rwthmoodle/moodle-mod_pdfannotator#41]
Clicking the _Overview_ tab withing the activity leads to the following error:

```
Debug info: SQLState: 42000<br>
Error Code: 8120<br>
Message: [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Column 'mdl_pdfannotator_annotations.page' is invalid in the select list because it is not contained in either an aggregate function or the GROUP BY clause.<br>

SELECT a.id as annoid, a.page, a.pdfannotatorid, p.name AS pdfannotatorname, p.usevotes, cm.id AS cmid, c.id as commentid, c.content, c.userid, c.visibility, c.timecreated, c.isdeleted, c.ishidden, SUM(vote) AS votes, COUNT(answ.id)-1 AS answercount, MAX(answ.timecreated) AS lastanswered FROM mdl_pdfannotator_annotations a JOIN mdl_pdfannotator_comments c ON c.annotationid = a.id JOIN mdl_pdfannotator p ON a.pdfannotatorid = p.id JOIN mdl_course_modules cm ON p.id = cm.instance LEFT JOIN mdl_pdfannotator_votes v ON c.id=v.commentid LEFT JOIN mdl_pdfannotator_comments answ ON answ.annotationid = a.id WHERE c.isquestion = 1 AND p.course = '28' AND cm.id = '487' AND c.solved = 0 GROUP BY a.id, p.name, p.usevotes, cm.id, c.id
[array (
0 => '28',
1 => 487,
)]
Error code: dmlreadexception
Stack trace:

    line 486 of /lib/dml/moodle_database.php: dml_read_exception thrown
    line 324 of /lib/dml/sqlsrv_native_moodle_database.php: call to moodle_database->query_end()
    line 431 of /lib/dml/sqlsrv_native_moodle_database.php: call to sqlsrv_native_moodle_database->query_end()
    line 900 of /lib/dml/sqlsrv_native_moodle_database.php: call to sqlsrv_native_moodle_database->do_query()
    line 976 of /lib/dml/sqlsrv_native_moodle_database.php: call to sqlsrv_native_moodle_database->get_recordset_sql()
    line 739 of /mod/pdfannotator/locallib.php: call to sqlsrv_native_moodle_database->get_records_sql()
    line 167 of /mod/pdfannotator/controller.php: call to pdfannotator_get_questions()
    line 79 of /mod/pdfannotator/view.php: call to require_once()
```

Maybe the most important fact is that our Moodle platform runs with Microsoft SQL Server as database.

**System**

* Moodle 3.9.3+ (Build: 20201127)
* **DB:** Microsoft SQL Server
* **Plug-in:** PDF Annotator v1.4 release 4 (Build: 2020081200)

**Steps to reproduce**

1. Set up Moodle with SQL Server
2. Install _PDF Annotation Tool for Moodle_
3. Within a course add the activity _PDF Annotation_
4. In the _Adding a new PDF Annotation to…_ form enter a name and select an arbitrary PDF document from your computer
5. Click _Save and return to course_
    a) While viewing the PDF document…
6. Click the _Overview_ tab

**Screenshot**

![image](https://user-images.githubusercontent.com/486983/101181173-dd1cd200-364c-11eb-97ed-653807b3e66a.png)

**Comment by Friederike-cil** [comment]
this should be fixed in v1.4.5

**Comment by mrkskwsnck** [comment]
I confirm the fix is working. Thank you very much!
